lib/validators/suppressions.ts

```diff
@@ -5,14 +5,15 @@ import * as path from "path"
 import * as md from "@ts-common/commonmark-to-markdown"
 import * as amd from "@ts-common/azure-openapi-markdown"
 import { isArray } from "@ts-common/iterator"
+import * as vfs from "@ts-common/virtual-fs"
 
 export const getSuppressions = async (specPath: string): Promise<undefined | amd.Suppression> => {
   // find readme.md
   const readMe = await amd.findReadMe(path.dirname(specPath))
   if (readMe === undefined) {
     return undefined
   }
-  const readMeStr = await amd.readFile(readMe)
+  const readMeStr = await vfs.readFile(readMe)
```

Comment:
is vfs stuff required to fix the issue that we're seeing with paths not valid in portal? do we need to introduce this now? we haven't reviewed or validated any of the vfs code

Reply:
`virtual-fs` is a copy from `azure-openapi-markdown`. @billytrend reviewed the `azure-openapi-markdown` 

- introducing VFS in `azure-openapi-markdown`
https://github.com/ts-common/azure-openapi-markdown/pull/11/files

- move it to separate lirary
https://github.com/ts-common/azure-openapi-markdown/pull/13

`vfs` should not be a part of `azure-openapi-markdown`.

Reply:
@veronicagg BTW, `virtual-fs` has 100% test coverage.